src/Dialect/ONNX/ONNX.td

```diff
@@ -217,6 +217,25 @@ def ONNXNoneOp : ONNX_Op<"NoValue", [ConstantLike, NoSideEffect]> {
     }]>];
 }
 
+def ONNXDimOp : ONNX_Op<"Dim", [NoSideEffect, DeclareOpInterfaceMethods<ShapeInferenceOpInterface>]> {
```

Comment:
I understand the convenience of using the ONNX dialect both for the standard ops as well as ops that are useful for us to perform some optimizations.

Would it make sense to label ops that we add for convenience (and I understand this is not the first one) as "Helper Ops" in the description? What do you think. As we add more of them, it might be nice to have a clear marker of the ones we added.

Reply:
Totally agree. Will add some markers for these ops.

Reply:
I added this sentent at the beginning of the description: `This operation is defined by onnx-mlir and not in the ONNX specification.`